force-app/main/default/layouts/Contact-Speaker Layout.layout-meta.xml

```diff
@@ -8,6 +8,10 @@
         <editHeading>true</editHeading>
```

Comment:
@bmtemakina I believe this layout is related to Contact object, is that correct ? if so please remove this metadata form this PR and create another PR for this change.